cluster-autoscaler/processors/status/scale_up_post_processor.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
+
+import (
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/cluster-autoscaler/context"
+)
+
+// ScaleUpStatus is the status of a scale-up attempt. This includes information
+// on if scale-up happened, description of scale-up operation performed and
+// status of pods that took part in the scale-up evaluation.
+type ScaleUpStatus struct {
+	ScaledUp                bool
+	ScaleUpDescription      []string
```

Comment:
Why not make this ScaleUpInfo? You end up converting them to string specifically so you can pass them to processor. You can do it in processor just as well and having structured information feels more useful than a bunch of strings (in a format designed for human reader, not parsing).

Reply:
Done.